app-backup/optar/optar-1.0.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Codec for encoding data on paper"
+HOMEPAGE="https://github.com/colindean/optar"
+SRC_URI="https://github.com/colindean/${PN}/archive/${PV}-colindean.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+pgm2ps"
+
+S="${WORKDIR}/${P}-colindean"
+
+DEPEND="media-libs/libpng:*"
```

Comment:
`:*` is not correct here since slots other than `0` don't carry headers, and you're apparently building from source.